### PR TITLE
Add test cases for some weird issues with Event content type

### PR DESCRIPTION
**Pull request by hvelarde**
# Test only; do not merge!

* https://github.com/plone/plone.api/issues/364
* https://github.com/plone/plone.app.contenttypes/issues/465

### CODE REVIEW

**Comment by hvelarde** [comment]
@idgserpro I tested both weird issues we are having in collective.cover and was not able to reproduce them here; I think we have a problem in our test suite setup and we would need some help.

**Comment by hvelarde** [comment]
@idgserpro any feedback?

**Comment by idgserpro** [comment]
@hvelarde are you sure that the ```plone.api``` tests are run on the coredev of ```Plone 4.3```? It's not in [tests.cfg](https://github.com/plone/buildout.coredev/blob/4.3/tests.cfg).

And the travis buildout is not working.

**Comment by idgserpro** [comment]
@hvelarde see #367

**Comment by hvelarde** [comment]
@idgserpro I tested locally in my own machine and tests are passing; you can do the same.

**Comment by idgserpro** [comment]
@hvelarde I run the tests locally and they really passed.

It looks like #364 has been fixed. About the dx problem I do not know much.

Do these two problems occur in cover?

**Comment by hvelarde** [comment]
yes they do, and that's why I think we have a problem with the test fixture.

File: src/plone/api/tests/test_content.py [hunk 1]
```diff
@@ -376,6 +376,46 @@ def test_create_at_with_title_in_request(self):
 
         self.assertEqual(page.title, 'Test document')
 
+    @unittest.skipIf(HAS_PACONTENTYPES, 'Archetypes only')
+    def test_create_at_event(self):
```
Comment (idgserpro) [inline src/plone/api/tests/test_content.py]: @hvelarde why do not you add this test to the package? It will prevent #364  from occurring again.

Comment (hvelarde) [reply src/plone/api/tests/test_content.py]: Archetypes is going to die soon.

File: src/plone/api/tests/test_content.py [hunk 2]
```diff
@@ -376,6 +376,46 @@ def test_create_at_with_title_in_request(self):
...
+        self.assertEqual(event.startDate, today)
+        self.assertEqual(event.endDate, tomorrow)
+        results = api.content.find(Title=u'My event')
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0].start, today)
+        self.assertEqual(results[0].end, tomorrow)
+
+    @unittest.skipUnless(HAS_PACONTENTYPES, 'Dexterity only')
+    def test_create_dx_event(self):
+        """https://github.com/plone/plone.app.contenttypes/issues/465"""
```
Comment (idgserpro) [inline src/plone/api/tests/test_content.py]: @hvelarde  what's the relationship between plone/plone.app.contenttypes#465 and ```plone.api```. It seems to me more a problem of ```plone.app.contenttype```.  What's purpose of this test?

@gbastien FYI.

**Pull request closed without merging**